app/views/layouts/application.html.erb

```diff
@@ -6,11 +6,15 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'application', media: 'all' %>
+    <%= javascript_pack_tag 'application' %>
```

Comment:
Why did you decide to disable turbolinks? Unless you're planning to use a lot of client-side rendering with JS (usually with React or some JS framework), Turbolinks will usually improve performance.

Reply:
So I'm planning on using Hotwire and I was under the impression that Turbolinks messes with Hotwire. But doing a quick search, I can't seem to find proof that it messes with Hotwire, only that it's preferable to disable it.

Reply:
Oh nice! I haven't tried using Hotwire before, but I think I also heard that it doesn't play nice with Turbolinks.